selftests/run

```diff
@@ -3,54 +3,17 @@
 
 __author__ = 'Lucas Meneghel Rodrigues <lmr@redhat.com>'
 
-import gc
```

Comment:
Not only that, but `selftests/run` is not our primary entry point for running tests (`selftests/check` is).

Reply:
Are you suggesting removing this file entirely ?